_posts/2016-07-10-chocolate-bread-pudding.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  Chocolate Bread Pudding
+author: Pomax
+date:   2016-07-10 8:10:42 -0800
+categories: vegetarian
+image: chocolate-bread-pudding.jpg
+time: 1.5 hr + overnight cooling
+ingredients:
+  - stale bread (100%)
+  - milk (150%) 
+  - wipping cream (150%)
+  - 1 large eggs per 150gr of bread
+  - plain white sugar (50%)
+  - proper brown sugar, not demerara (25%)
+  - dark chocolate (150%)
+  - a tablespoon of butter
+  - a good splash of vanillin
+  - a pinch of ground clove
+  - quite a bit of ground cinnamon
+allergy_info:
+  - milk, egg, gluten
```

Comment:
I most definitely can, but note that a wheat allergy and a gluten allergy are not the same thing =) (I have some relatives who have wheat-but-not-gluten sensitivities)